/*
 * Copyright (c) 2014 Juniper Networks, Inc. All rights reserved.
 */

import "google/protobuf/descriptor.proto";

message SelfDescribingMessageTest {
  // Set of .proto files which define the type.
  required google.protobuf.FileDescriptorSet proto_files = 1;

  // Name of the message type.  Must be defined by one of the files in
  // proto_files.
  required string type_name = 2;

  // The message data.
  required bytes message_data = 3;
}
